odoo/tools/misc.py

```diff
@@ -802,6 +809,44 @@ def deco(*args, **kwargs):
                 return func(*args, **kwargs)
         return deco
 
+    def emit(self, record):
+        pass
+
+
+class quiet_logger(logging.Handler):
+    """Temporary lower the max logging level.
```

Comment:
Only really works if there are no handlers outside the root loggers though.

Reply:
Good enough for us right now.